app/src/forms/email/emailService.js

```diff
@@ -0,0 +1,142 @@
+const chesService = require('../../components/chesService');
+const fs = require('fs');
+const log = require('npmlog');
+const path = require('path');
+
+const formService = require('../form/service');
+
+const service = {
+
+  _appUrl: (referer) => {
```

Comment:
Typo: referrer, not referer (happens in multiple places)

Reply:
https://en.wikipedia.org/wiki/HTTP_referer